sitemedia/js/controllers/ittpanel_controller.js

```diff
@@ -28,5 +46,86 @@ export default class extends Controller {
                     break;
             }
         }
+
+        if (this.isDesktop()) {
+            if (this.transcriptionAndTranslationOpen()) {
+                // when transcription and translation are both opened, align their contents line-by-line
+                this.alignLines();
+            } else {
+                // when one of those two toggles is closed, remove inline styles from lines (alignment no longer needed)
+                this.transcriptionTarget
+                    .querySelectorAll("li")
+                    .forEach((li) => {
+                        li.removeAttribute("style");
+                    });
+                this.translationTarget.querySelectorAll("li").forEach((li) => {
+                    li.removeAttribute("style");
+                });
+            }
+        }
+    }
+
+    transcriptionAndTranslationOpen() {
+        // check toggle targets to find out whether both transcription and translated are open
+        return ["transcription-on", "translation-on"].every(
+            (id) =>
+                this.toggleTargets.find((target) => target.id === id)
+                    .checked === true
+        );
+    }
+
+    alignLines() {
+        // align each line of transcription to translation
+        const edLines = this.transcriptionTarget.querySelectorAll("li");
+        const trLines = this.translationTarget.querySelectorAll("li");
+        // only align as many lines as we need to
+        const [lessLines, moreLines] =
+            edLines.length < trLines.length
+                ? [trLines, edLines]
+                : [edLines, trLines];
+        lessLines.forEach((li, i) => {
+            if (li && moreLines[i]) {
+                // compare top of lines by position in list
+                const thisLiPos = li.getBoundingClientRect();
+                const thatLiPos = moreLines[i].getBoundingClientRect();
+                // add padding-top to the elements that need it
+                if (thisLiPos.top < thatLiPos.top) {
+                    li.style.paddingTop = `${thatLiPos.top - thisLiPos.top}px`;
+                } else if (thisLiPos.top > thatLiPos.top) {
+                    moreLines[i].style.paddingTop = `${
+                        thisLiPos.top - thatLiPos.top
+                    }px`;
+                }
```

Comment:
> Is top padding basically accounting for the space of the previous line rather than the corresponding line?
Essentially, yes, but it's actually using the y-position of the top of the corresponding line to determine how much padding to add.

I like this approach for its cleanliness but not sure it's worth the effort to change, depending on how you feel about my below comment on sanitization.